winui/Ribbon/Ribbon-QAT.md

```diff
@@ -121,6 +122,22 @@ The Ribbon also supports adding items to the QAT Menu items. To add items to the
 
 ![WinUI Ribbon QAT menu items](Ribbon-QAT-images/winui-ribbon-qat-menu-items.png)
 
+## Adding items using QAT Customize dialog window
+
+To open QAT customize dialog window, select `MoreCommands` option from the ContextMenu of the QAT. This dialog window showcases all available commands within the Ribbon control.
+
+By using `Choose commands from` option, you can filter the command based on ribbon tabs. Then, select the desired command and click `Add` button to add the command to the right Pane of the QAT dialog window. Finally, click `OK`.
+
+![MoreCommands option in QAT drop-down context menu](Ribbon-QAT-images/winui-ribbon-more-commands-option.png)
+
+![Select commands in QAT dialog window](Ribbon-QAT-images/winui-ribbon-select-command-in-qat-dialog-window.png)
+
+![Adding command to right pane of QAT dialog window](Ribbon-QAT-images/winui-ribbon-add-command-to-right-pane-in-qat-dialog-window.png)
+
+Finally, the Items gets displayed in the QAT
```

Comment:
Finally, the Items are displayed in the QAT.

Reply:
modified